components/webui/package.json

```
@@ -2,21 +2,26 @@
   "name": "webui",
   "version": "0.1.0",
   "description": "",
+  "workspaces": [
+    "common",
+    "client",
+    "server"
```

would it be better to put those under a `packages/` directory? this seems to be the recommendation in the npm docs and a commonly used conventions in many projects.
```suggestion
    "packages/common",
    "packages/client",
    "packages/server"
```